src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -168,7 +171,15 @@ public void scenario1() throws IOException {
         setAuth(request, "smith123");
         try (final CloseableHttpResponse response = execute(request)) {
             assertEquals(HttpStatus.SC_OK, getStatus(response));
+            for (final Header header : response.getHeaders("Link")) {
+                logger.debug("Link header: {}", header.getValue());
+                if (header.getValue().equals(aclLink)) {
+                    return;
+                }
+            }
+            assertEquals("Missing Link Header " + aclLink.toString(), 1, 0);
```

Comment:
@acoburn but is a for loop the best way to do this type of testing? It seems very basic, but perhaps I am just expecting more.